content/chapters/software-stack/lecture/slides/software-stack.md

```diff
@@ -0,0 +1,91 @@
+---
+
+## Software Stack
+
+---
+
+### Flexibility of Software
+
+TODO: Diagram with users, hardware and software
+
+----
+
+### Flexibility of Software
+
+* Hardware provides basic actions
+* Users request rich features
+* Software bridges the gap between user demands and hardware possibility
+
+----
```

Comment:
Maybe add a slide about the efficiency of hardware? It would be a good contrast: efficiency at the cost of flexibility, which is an upside of software.

Reply:
That will be in the table on the comparison slide.